docs/user/dashboard/lens.asciidoc

```diff
@@ -31,42 +31,23 @@ Lens has a fully accessible, continuously improved drag and drop system. By usin
 [role="screenshot"]
 image::images/lens_drag_drop_1.png[Presented Lens drag and drop]
 
-[float]
-[[drag-and-drop-keyboard-navigation-drop-a-field]]
-==== Drop a field to the workspace or configuration panel
-
-. Set a focus on the chosen field. 
-. Press space to start a drag.
-. Using left and right arrows, select a drop target. 
-. Press space to confirm.
-
-[float]
-[[drag-and-drop-keyboard-navigation-duplicate-an-operation]]
-==== Duplicate an existing operation
-
-Creating a few similar operations can be repetetive and time-consuming. A good usecase would be a need for creating multiple percentiles. Instead of configuring each metric from scratch, you can duplicate it – just drag a metric to duplicate and drop it into an empty dimension. For keyboard navigation:
-
-. Set focus on an item you want to duplicate.
-. Press space to start a drag.
-. Using left and right arrow keys, choose as a target a button `Drop a field or click to add` in the same group.
-. Confirm with space.
+Follow the instructions below to take advantage of drag and drop keyboard navigation:
 
+. Set a focus on the chosen item. Most of draggable elements have two focus states. One serves to open a panel with more detailed information or options and another one to drag an element. To drag an element, tab through the page until the focus state looks like on the picture:
```

Comment:
This is definitely more clear than before, but are we actually able to say "looks like the picture" in docs? Maybe we can use more descriptive language to say that the two focus states are "outer" and "inner" or something like that?

Reply:
I know, I definitely agree, it's not a very accessible language. I was struggling a lot with it and then decided to leave it with my first thought and count of feedback 😅 I'll see what Kaarina thinks, so far your proposal is better than mine.